src/cdk-experimental/menu/menu-item-checkbox.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive} from '@angular/core';
+import {CdkMenuItemSelectable} from './menu-item-selectable';
+
+/**
+ * Directive which provides simple checkbox behavior to an element it is applied to.
+ */
+@Directive({
+  selector: '[cdkMenuItemCheckbox]',
+  exportAs: 'cdkMenuItemCheckbox',
+  host: {
+    '(click)': 'trigger()',
+    'type': 'button',
+    'role': 'menuitemcheckbox',
+    '[attr.aria-checked]': 'checked',
```

Comment:
Since it's a checkbox wouldn't it be explicitly checked or not?

Reply:
We've heard some feedback that making elements `null` when they would be `false` makes the screen-reader less verbose, so rather then being e.g.
```
Cheese selected
Tomato not selected
Onion not selected
Pepper selected
```
it would instead read
```
Cheese selected
Tomato
Onion
Pepper selected
```
So that "not selected" is implicit